rd_ui/app/scripts/directives/destination_directives.js

```diff
@@ -0,0 +1,80 @@
+(function () {
+  'use strict';
+
+  var directives = angular.module('redash.directives');
+
+  directives.directive('destinationForm', ['$http', 'growl', function ($http, growl) {
```

Comment:
We should have a single component (directive) of "DynamicForm", that will be used both here and in the data source form, to reduce the code duplication.

Will you need help with this?